### PR TITLE
[Accessibility] Drawing Bluetooth pattern WIP

### PR DESCRIPTION
**Pull request by microbit-grace**
https://microbit-global.monday.com/boards/1356069004/pulses/1368253368

WIP!!

### CODE REVIEW

**Comment by microbit-robert** [comment]
For some reason the bottom right LED cannot be hovered / highlighted directly or indirectly:
image

**Comment by microbit-robert** [comment]
- The inputs don't stop a user typing 100 or 0 and then tabbing along to the next input.
- Should we use `cursor: pointer;`?
- Should the highlight work on increasing and decreasing the number of lit LEDs. It only has changes for increasing at the moment. Not sure if this is really for this PR or a follow up.

Overall it works well and makes it accessible.

File: src/messages/ui.en.json [hunk 1]
```diff
@@ -183,6 +183,7 @@
 
   "connectMB.pattern.heading": "Copy pattern",
   "connectMB.pattern.subtitle": "Copy the pattern displayed on the micro:bit.",
+  "connectMB.pattern.inputLabel": "Number of lit up LED in column {colNum} displayed on micro:bit",
```
Comment (microbit-robert) [inline src/messages/ui.en.json:186]: Will wait for Lucy to comment, but maybe this is more like "The height of illuminated LEDs in column {colNum} on the micro:bit"?

**Comment by microbit-grace** [comment]
> For some reason the bottom right LED cannot be hovered / highlighted directly or indirectly
> The inputs don't stop a user typing 100 or 0 and then tabbing along to the next input.

Great catches!  

> Should we use cursor: pointer;?

Good point, yes, I think we should to make the inputs completely undetectable when unfocused

> Should the highlight work on increasing and decreasing the number of lit LEDs. It only has changes for increasing at the moment. Not sure if this is really for this PR or a follow up.

Will see whether this PR touches that logic, if so, can do in this PR :)

**Comment by microbit-grace** [comment]
Will work towards merging for now, and do potential copy changes later

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**